src/gpu/generic/sycl/ref_softmax.cpp

```diff
@@ -36,11 +36,29 @@ status_t ref_sycl_softmax_fwd_t::pd_t::init_conf() {
     conf_.outer_size = outer_size();
     conf_.channels = axis_size();
     conf_.wk_size = inner_size() * outer_size();
+
+    for (int i = 0; i < DNNL_MAX_NDIMS; i++) {
+        conf_.dst_dims[i] = dst_md()->dims[i];
+    }
+    conf_.dst_ndims = dst_md()->ndims;
+
     conf_.do_scale_src
             = !attr()->scales_.get(DNNL_ARG_SRC).has_default_values();
     conf_.do_scale_dst
             = !attr()->scales_.get(DNNL_ARG_DST).has_default_values();
 
+    const auto *att = attr();
```

Comment:
Why are you storing that into a variable if you are then still calling `attr()` in other places? Either use variable everywhere or remove it.

Removing it would be more consistent with other primitives I think.

Reply:
Done.